cmd/opentelemetry/go.mod

```diff
@@ -4,12 +4,13 @@ go 1.13
 
 replace k8s.io/client-go => k8s.io/client-go v0.0.0-20190620085101-78d2af792bab
 
-replace github.com/jaegertracing/jaeger => ./../../
+// TODO uncomment once OTEL upgrades Jaeger with thrift 0.13
+//replace github.com/jaegertracing/jaeger => ./../../
```

Comment:
I had to do this otherwise Jeger OTEL would not compile - OTEL depends on older Jaeger version using older thrift.

Reply:
please book a ticket